templates/tfengine/components/resources/healthcare_datasets/main.tf

```diff
@@ -103,9 +103,11 @@ module "{{resourceName . "name"}}" {
             schema_config = {
               recursive_structure_depth = "{{$v.bigquery_destination.schema_config.recursive_structure_depth}}"
               schema_type = "{{$v.bigquery_destination.schema_config.schema_type}}"
+              {{if has . "last_updated_partition_config" -}}
```

Comment:
I dont think this work because . refers to the root level data object, but you want to look for within v. I think you can replace the whole block with 

`{{hclsField $v.bigquery_destination.schema_config "last_updated_partition_config" -}}